Configs/.config/fish/config.fish

```diff
@@ -1,22 +1,33 @@
+# Disable the default Fish greeting
 set -g fish_greeting
 
+# Initialize Starship prompt if interactive
 if status is-interactive
     starship init fish | source
 end
 
-# List Directory
-alias l='eza -lh  --icons=auto' # long list
-alias ls='eza -1   --icons=auto' # short list
-alias ll='eza -lha --icons=auto --sort=name --group-directories-first' # long list all
-alias ld='eza -lhD --icons=auto' # long list dirs
-alias lt='eza --icons=auto --tree' # list folder as tree
+# Directory Listing Aliases with eza
+alias l='eza -lh  --icons=auto'  # Long list with icons
+alias ls='eza -1   --icons=auto'  # Short list with icons
+alias ll='eza -lha --icons=auto --sort=name --group-directories-first'  # Long list all with icons
+alias ld='eza -lhD --icons=auto'  # Long list dirs with icons
+alias lt='eza --icons=auto --tree'  # List folder as tree with icons
 
-# Handy change dir shortcuts
+# Handy change directory shortcuts
 abbr .. 'cd ..'
 abbr ... 'cd ../..'
 abbr .3 'cd ../../..'
 abbr .4 'cd ../../../..'
 abbr .5 'cd ../../../../..'
 
-# Always mkdir a path (this doesn't inhibit functionality to make a single dir)
+# Enhanced clear command
+function clear --description 'Clear the screen and scrollback buffer'
+    command clear
+    printf '\e[3J'
```

Comment:
Do we really need a function for `clear `?

Reply:
I added that function for a cleaner `clear` command. I believe this improves readability and maintains consistency in my setup. It's not strictly necessary, but it helps in streamlining the process and keeping the configuration tidy.